butterfree/load/writers/writer.py

```diff
@@ -50,3 +54,41 @@ def validate(
             AssertionError: if validation fails.
 
         """
+
+    def build(
+        self, feature_set: FeatureSet, dataframe: DataFrame, spark_client: SparkClient,
+    ) -> Optional[StreamingQuery]:
+        """Trigger the writer from a feature set into the Feature Store.
+
+        Args:
+            feature_set: object processed with feature set metadata.
+            dataframe: Spark dataframe containing data from a feature set.
+            spark_client: client for Spark connections with external services.
+
+        If the debug_mode is set to True, a temporary table with a name in the format:
+        {feature_set.name} will be created instead of writing
+        to the real historical or online feature store.
+        If dataframe is streaming this temporary table will be updated in real time.
+
+        """
+        pre_hook_df = self.run_pre_hooks(dataframe)
+
+        if self.debug_mode:
+            return self._write_in_debug_mode(
+                table_name=f"{feature_set.name}",
```

Comment:
One attention point that can be solved later, we could create an issue to think about that, no biggie. 

Here I think we'll have a problem when using more them one writer in debug mode since the view will be overwritten with the same name. One way would be creating the name using the `__name__` property on the writers as a prefix (maybe is needed to properly format like downcase, changing spaces for underscore). Or we could create each temp table with a timestamp, so they don't overwrite each other.

Another thing is that we could log for the user the name of the created table. Some users can't find the temp tables they create haha.